### PR TITLE
260 optionspage linked

### LINKED ISSUE #260: Optionspage linked
There is an option page....  #123 But when I click on "Options/ Protected Page" I dont get redirected to this page. Please rename "Protected Page" to "Options" with the same symbol that we have on mobile. Please make  "Logout" and "Edit Personal Informations" work. The functionality exists. Only these 2 buttons on the options page, ignore the other buttons on the mock below.

Screenshot 2022-02-04 at 15 56 14

### PR DESCRIPTION
**Pull request by treyfel**
Closes #260 .
Optionspage now looks like this:
image

### CODE REVIEW

File: app/views/options/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<% content_for :title, "Map" %>
```
Comment (kpostnov) [inline app/views/options/index.html.erb:2]: Does this line make sense?

**Comment by kpostnov** [comment]
Maybe, since we are already working on the options page, you could a link to the sign in button as well?

File: app/views/options/index.html.erb (hunk 1 above)
Comment (treyfel) [reply app/views/options/index.html.erb:2]: No clue what this does

**Comment by treyfel** [comment]
If you're not signed in:
image
if you click on signed in youre redirected to the login page

**Comment by treyfel** [comment]
Navbar in webview now has optionspage
image

**Comment by kpostnov** [comment]
Ok, looks good.
One last small thing: can you change the icon in navbar as well to make it look like in the bottombar?

**Comment by treyfel** [comment]
sure :)

**Review by kpostnov — APPROVED** [review]

**Pull request merged**